### DISCUSSION TITLE
Public Roadmap?

### DISCUSSION THREAD

**Discussion by raphi011** [zed-industries/zed#7212] [Feature Requests]
Hey :) First of all thanks for the great work on Zed - I'm trying to move to Zed as my primary editor because i really value responsive uis and I don't want to go down rabbit hole of e.g. configuring vim .. 

I noticed that one of the latest zed [blog posts](https://zed.dev/blog/zed-is-now-open-source) mentioned a roadmap 

> ... and we've put together a public roadmap based on user feedback ...

But I haven't found it anywhere? Can you post a link to it? It would be nice to see which new features are coming up in the near future!

**Comment by bennetbo** [comment]
There is a roadmap on the zed website (not sure if its up to date): https://zed.dev/roadmap

**Reply by raphi011** [reply]
nice! that's probably the one, maybe it could be linked in the blog post :) thanks!

**Reply by Lukas-Heiligenbrunner** [reply]
I get a 404 Page not found for this roadmap url, you guys too?

**Reply by zytact** [reply]
Me too. Maybe they took down the page?
